include/nbl/system/DefaultFuncPtrLoader.h

```diff
@@ -24,7 +24,7 @@ class DefaultFuncPtrLoader final : FuncPtrLoader
 {
 	protected:
 		#if defined(_NBL_WINDOWS_API_)
-			HINSTANCE lib;
+			HMODULE lib;
```

Comment:
O RLY?
I got the type wrong?

Reply:
yeah, theoretically, they're all void ptrs or something in the end but yeah